disperser/batcher/txn_manager.go

```diff
@@ -141,21 +141,38 @@ func (t *txnManager) ProcessTransaction(ctx context.Context, req *TxnRequest) er
 	t.mu.Lock()
 	defer t.mu.Unlock()
 	t.logger.Debug("new transaction", "component", "TxnManager", "method", "ProcessTransaction", "tag", req.Tag, "nonce", req.Tx.Nonce(), "gasFeeCap", req.Tx.GasFeeCap(), "gasTipCap", req.Tx.GasTipCap())
-	gasTipCap, gasFeeCap, err := t.ethClient.GetLatestGasCaps(ctx)
-	if err != nil {
-		return fmt.Errorf("failed to get latest gas caps: %w", err)
-	}
 
-	txn, err := t.ethClient.UpdateGas(ctx, req.Tx, req.Value, gasTipCap, gasFeeCap)
-	if err != nil {
-		return fmt.Errorf("failed to update gas price: %w", err)
+	var txn *types.Transaction
+	var txID walletsdk.TxID
+	var err error
+	retryFromFailure := 0
+	for retryFromFailure < maxSendTransactionRetry {
+		gasTipCap, gasFeeCap, err := t.ethClient.GetLatestGasCaps(ctx)
```

Comment:
What's the main reason that we fail on the ethClient errors and retry on the wallet error? Couldn't the `GetLatestGasCaps` call timeout?

Reply:
It could timeout, but the retry logic is built into the ethClient we use (`MultiHomingClient`).